pod/perlfunc.pod

```diff
@@ -9572,6 +9572,17 @@ the string.  A positive OFFSET greater than the length of SCALAR
 results in the string being padded to the required size with C<"\0">
 bytes before the result of the read is appended.
 
+    open(my $FH, "<", "input.txt") or die("Cannot open file: $!");
+
+    my $buf = "";
+    my $num = 0;
+
+    # Read 64 bytes
```

Comment:
"Read up to 64 bytes" would be more accurate. Unlike read it doesn't retry if it can't fill the buffer in one go.

Reply:
Good to know. PR updated